src/js/components/Carousel/propTypes.js

```diff
@@ -1,31 +1,42 @@
 import PropTypes from 'prop-types';
 import { genericProps } from '../../utils/general-prop-types';
 
-let PropType = {};
+let CarouselPropType;
 if (process.env.NODE_ENV !== 'production') {
-  PropType = {
+  CarouselPropType = {
     ...genericProps,
     activeChild: PropTypes.number,
-    fill: PropTypes.bool,
-    play: PropTypes.number,
     initialChild: PropTypes.number,
+    fill: PropTypes.bool,
+    controls: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
Should we accept any strings or enumerate the accepted values?

Reply:
If we revert to 
```
controls: PropTypes.oneOfType([
      PropTypes.bool,
      PropTypes.oneOf(['arrows', 'selectors']),
    ]),
```

then I don't think we need to touch `index.d.ts` -- otherwise we'll want these types to be in sync.

Reply:
Right, I can add those values back in.